tests/test_stubs.py

```diff
@@ -1,34 +1,59 @@
-import os.path
+import os
+from pathlib import Path
 import pytest
 from mypy import api
 
 
-TESTS_DIR = os.path.dirname(__file__)
+TESTS_DIR = Path(__file__).parent
 
 
 def gen_tests():
-    for filename in os.listdir(TESTS_DIR):
-        if filename.endswith('.py') and not filename.startswith('test_'):
-            yield filename
+    """List of all tests files included in the directory tests"""
+    for path in TESTS_DIR.glob('*.py'):
+        if not path.name.startswith('test_'):
+            yield path
```

Comment:
```suggestion
    yield from TESTS_DIR.glob('test_*.py')
```

Reply:
the suggestion changes the code. We want the path not starting with test_

Reply:
Uh, right...  Sorry.